### ISSUE TITLE
Fixing the minor issues in the calculator, including the position of the resulting variable

### ISSUE THREAD

**Issue by lilyclements** [IDEMSInternational/R-Instat#6057]
The name of this issue is now more general to indicate other improvements made while this issue is fixed.

There is a very small bug with the `TestOKEnabled` in the calculator dialog where OK is enabled if there is no name given in the ucrSave for the new column. Currently the sub aimed at this is:

```
Private Sub ucrCalc_SaveNameChanged() Handles ucrCalc.SaveNameChanged
        SaveResults()
        TestOKEnabled()
End Sub
```

However this does not run if you remove the name of the new column. This means that you can press OK without giving a name for the new save column.
If you reopen the dialog, OK is disabled. It does not then re-enable if you add in a new column name (until you remove and add a name in the Expression box, or click Reset).

**Comment by N-thony** [comment]
@rdstern Can this issue be taken or is it still under discussion?

**Comment by rdstern** [comment]
I hope this is being done by @Patowhiz as he has another change to make on this dialogue.  Good if you can check once he does that.

**Comment by N-thony** [comment]
> I hope this is being done by @Patowhiz as he has another change to make on this dialogue. Good if you can check once he does that.

@rdstern thanks for letting me know. @Patowhiz did you fixed this issue? Thanks

**Comment by Patowhiz** [comment]
@N-thony yes I'm working on this dialog.

**Comment by rdstern** [comment]
@Patowhiz is this going to be fixed and closed when the main calculator pull request is merged?

**Comment by Patowhiz** [comment]
@rdstern yes. The event was not being raised in the ucrCalculator. That is now fixed in the PR.
